css/form.css

```diff
@@ -62,3 +62,22 @@ textarea:focus {
 .contact-me-form-button {
   align-self: flex-start;
 }
+
+@media screen and (min-width: 768px) {
+  .contact-me-form-container {
+    align-content: center;
+    padding: 10% 25%;
+    background-image: url(./assets/contact-form-shapes-desktop.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+  }
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that the background image used in the Figma design should be visible in your portfolio. Please verify the path used to ensure that your design sticks to the Figma design 👍 .
- [ ] [OPTIONAL] You have done incredibly well to implement the changes. I add on to suggest that you ensure that the `border-radius` used on the inputs of your portfolio is the same as that of the Figma template design.

Your design 😎 
![contact](https://user-images.githubusercontent.com/99394374/215256298-79ba2473-42b3-4cd9-bc70-4497697349be.PNG)

Figma design 
![contact fig](https://user-images.githubusercontent.com/99394374/215256353-68795d41-e134-40b5-88fa-9149057a15c0.PNG)